roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1214,8 +1261,10 @@ def set_container_facts_if_unset(facts):
     if safe_get_bool(facts['common']['is_containerized']):
         facts['common']['admin_binary'] = '/usr/local/bin/oadm'
         facts['common']['client_binary'] = '/usr/local/bin/oc'
-        base_version = get_openshift_version(facts, cli_image).split('-')[0]
-        facts['common']['image_tag'] = "v" + base_version
+        openshift_version = get_openshift_version(facts, cli_image)
+        if openshift_version is not None:
+            base_version = get_openshift_version(facts, cli_image).split('-')[0]
```

Comment:
probably better to reuse openshift_version here instead of calling get_openshift_version twice.